Trimble.Modus.Components/Controls/Button/CustomButton.xaml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Trimble.Modus.Components.Controls.Button.CustomButton"
+             x:Name="customButton"
+             xmlns:constants="clr-namespace:Trimble.Modus.Components.Contant"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters">
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <converters:ImageNullOrEmptyBoolConverter x:Key="ImageNullOrEmptyBoolConverter" />
+      <converters:InverseBoolConverter x:Key="InverseBoolConverter" />
```

Comment:
We should be using MAUI toolkit's converter whenever possible. If you can't use them right now, can you create a tech debt issue and link it here?

Reply:
Tried to use the MAUI toolkit. Faced some reference issues. Will create a ticket for this.

Reply:
https://github.com/trimble-oss/modus-mobile-maui-components/issues/113

MAUI community toolkit integration